CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - [v1.3.0-alpha.3](#v130-alpha3)
   - [Downloads](#downloads)
+- [v1.3.0-alpha.3](#v130-alpha3)
```

Comment:
> How did this happen?

I ran `hack/update-generated-docs.sh` which I thought I needed to do to update the munged TOC and this appeared.  I assumed the script knew what it was doing.  Should it come back out?